config/olm/addon-operator.csv.tpl.yaml

```diff
@@ -217,7 +217,25 @@ spec:
                 app.kubernetes.io/name: addon-operator
             spec:
               serviceAccountName: addon-operator
+              volumes:
+              - name: tls
+                secret:
+                  secretName: metrics-server-cert
               containers:
+              - name: metrics-relay-server
+                image: quay.io/brancz/kube-rbac-proxy:v0.11.0
+                args:
+                - "--secure-listen-address=0.0.0.0:8443"
+                - "--upstream=http://127.0.0.1:8080/"
+                - "--tls-cert-file=/tmp/k8s-metrics-server/serving-certs/tls.crt"
+                - "--tls-private-key-file=/tmp/k8s-metrics-server/serving-certs/tls.key"
+                - "--logtostderr=true"
+                - "--ignore-paths=/metrics,/healthz"
+                - "--v=10"
```

Comment:
Do we want to keep debug level logging for prod?

Reply:
whoops, forgot to tweak this after copying the dev manifests. On it!